lib/tasks/admin/create_admin.rake

```diff
@@ -0,0 +1,19 @@
+namespace :admin do
```

Comment:
create another rake task that takes in an email as command line argument and makes an existing user with that email admin

Reply:
Done